.github/workflows/test.yml

```diff
@@ -50,6 +50,9 @@ jobs:
           - name: "worker and utils"
             markers: "not e2e"
             flags: "--timeout=300"
+          - name: "quantized"
+            markers: "quantized"
```

Comment:
This would need to be `cpu and quantized` if we wanted to run on GHA

But this test would also already run in the `static batching` job as well, since it matches `cpu and decoder and not cb`. I would slightly prefer leaving it in the static batching job instead of creating anew job, just because we have so many entries in the test matrix now, and each one takes a couple minutes of CI worker time to spin up, load cache, and prepare the test environment